portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,51 @@
 #content {
+  margin-top: auto;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  padding: 20px;
+  max-width: 800px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#header-container {
+  margin-bottom: 20px;
 }
+
+/* top image hidden by default */
+.img-top {
+  display: none;
+}
+#avatar img{
+  max-height: 100px;
```

Consistent spacing :)
```suggestion
.img-top {
  display: none;
}

#avatar img{
  max-height: 100px;
```

---

I removed the empty line because those two css blocks style the same element of the html. But I guess the convention doesn't do that?